.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "singleQuote": true,
+    "trailingComma": "none",
```

Comment:
I would prefer trailing commas as often as possible: it leads to smaller diffs in the future, leading to fewer merge conflicts, and a better git blame.

Reply:
@vidartf I replaced option `"trailingComma": "none"` by `"trailingComma": "all"`, hoping it corresponds to what you require.